nemo/collections/asr/modules/rnnt.py

```diff
@@ -1101,8 +1103,8 @@ def input_types(self):
 
         return mytypes
 
-    def input_example(self):
-        decoder_example = self.decoder.input_example()
+    def input_example(self, max_batch=1, max_dim=256):
```

Comment:
8192

Reply:
This one actually should be 1, as per previous implementation ...